apps/src/aichat/views/presentation/ModelCardRow.tsx

```diff
@@ -16,6 +16,26 @@ const ModelCardRow: React.FunctionComponent<ModelCardRowProps> = ({
   titleIcon,
   expandedContent,
 }) => {
+  const getExpandedContentToDisplay = () => {
+    if (Array.isArray(expandedContent)) {
+      // Remove empty strings from the array.
+      const checkedExpandedContent = expandedContent.filter(
+        content => content.length !== 0
+      );
+      if (checkedExpandedContent.length === 0) {
+        return <p>No examples available</p>;
```

Comment:
minor: is this only used for the "Example Topics" entry? If so, should we just explicitly check that `keyName === 'exampleTopics'`, instead of checking if the items are an array?

Alternatively; if it's meant to be reusable (if we hypothetically have other multiline fields), then this is fine, but we'd probably want to abstract out the `No examples available` default text

Reply:
Yes, we have one other multiline field (Technical Info) which should not ever be blank. So still want to filter out empty strings. But I'll update to more generic text. 'Not available'. Thanks!